src/main/kotlin/io/bazel/kotlin/builder/utils/ArgMap.kt

```diff
@@ -19,61 +19,92 @@ package io.bazel.kotlin.builder.utils
 import java.io.File
 
 class ArgMap(private val map: Map<String, List<String>>) {
-    /**
-     * Get the mandatory single value from a key
-     */
-    fun mandatorySingle(key: String): String =
-        optionalSingle(key) ?: throw IllegalArgumentException("$key is not optional")
-
-    fun labelDepMap(key: String) = optional(key)?.asSequence()?.windowed(2, 2)?.map { it[0] to it[1] }?.toMap() ?: emptyMap()
-
-    fun optionalSingle(key: String): String? =
-        optional(key)?.let {
-            when (it.size) {
-                0 -> throw IllegalArgumentException("$key did not have a value")
-                1 -> it[0]
-                else -> throw IllegalArgumentException("$key should have a single value")
-            }
+  /**
+   * Get the mandatory single value from a key
+   */
+  private fun mandatorySingle(key: String): String =
+      optionalSingle(key) ?: throw IllegalArgumentException("$key is not optional")
+
+  private fun labelDepMap(key: String) = optional(key)?.asSequence()?.windowed(2,
```

Comment:
Ok, there's no universe in which this is a reasonable line-length reflow. :D

Reply:
Fixed. 

I'm not quite sure what the function is doing, though.